src/components/block/ProgramCard/ProgramCard.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import ProgramCard from "./ProgramCard";
+import img from "./sample.jpg";
```

Comment:
This is a bit minor, but do you think we could move this image into a subdirectory named `stories/`? This would help avoids us getting confused about which images are actually part of the component vs. which ones are just for demonstration purposes in the story. I think for now, we can just leave that directory in each component directory, but we may want to revisit later to consolidate all Storybook placeholders into one consistent place.